File: OpenMP/hello1millomp.f90
program hello
	implicit none
        include "omp_lib.h"
	character(len=16) :: greetings 
	integer :: a
	greetings = "Hello World!" 
	
	!$omp parallel
	!$omp do 
	! Hello World, one million times
	do a = 1, 1000000
	print *, greetings   
	end do
	!$omp end parallel
	
end program hello

File: OpenMP/hello1millomp.c
#include <stdio.h> 
#include  "omp.h"
int main(void) 
{ 
   char greetings[] = "Hello world!"; 
   int a;
   #pragma omp parallel
   #pragma for
   for ( a = 0; a < 1000000; a = a + 1 )
   	{
	printf("%s\n", greetings); 
   	}
return 0;
}
